### ISSUE TITLE
Aubess Smart socket 20A/EM no DPS in attributes

### ISSUE THREAD

**Issue by MSZ-Spyro** [rospogrigio/localtuya#1083]
## The problem
I am trying to add Aubess Smart socket 20A/EM to my home assistant using LocalTuya and it discovers it, it shows additional parameters which i choose from the list (similar to the two plugs that ale listed as compatibile 18,19,20) and add it as a switch with those additional parameters (dunno why the ID is chosen to bee 1, which is a bool ("true" for me)) and the switch works. It can be switched ON/OFF and it sees its proper state but attributes are not added and i cant access them anyhow. 

## Environment

- Localtuya version: 4.1.1 latest 
- Last working localtuya version (if known and relevant): none of which i know
- Home Assistant Core version: 2022.10.4 (in the meantime updated from 2022.7.3 which was the same)
- [] Are you using the Home Assistant Tuya Cloud component ?
- [] Are you using the Tuya App in parallel ?

## Steps to reproduce

1. Add new device via GUI
2. Choose proper one from the list
3. Add necessary parameters
4. Choose attributes
5. See there are no attributes attached to the switch.

## DP dump
```
INFO:localtuya:localtuya version 1.0.0
INFO:localtuya:Python 3.8.10 (default, Mar 15 2022, 12:22:08)
[GCC 9.4.0] on linux
INFO:localtuya:Using pytuya version '8.1.0'
INFO:localtuya:Detecting list of available DPS of device ccccccccccccccccc [192.168.1.76], protocol 3.3.
DEBUG:localtuya.pytuya:Sending command status (device type: type_0a)
DEBUG:localtuya.pytuya:paylod=b'{"gwId":"ccccccccccccccccc","devId":"ccccccccccccccccc"}'
DEBUG:localtuya.pytuya:DATA RECEIVED!
DEBUG:localtuya.pytuya:decode payload=b'9S\xd8\x1f\xc8\xb8}\xc0\x1c\xe9\x19\xf5\xa5"h\xa2\x19{E\xf7u\xaaO\xab\xf1J(P\x03z\x08\x89E \x7f\xe3\r7\xe4\\\xe8=\xd4\xcb\xce\xd3\xf1\xa6\x8c\x95\xbc\xa9\xa2h-\x85\x90X\xe6\xaa\x19\xed\xf2\\\x93\xec\xae\xbd\xceS\xff\xc5\xb4+\xd8\x07t>^(\xceg \x91\x0fza\x03\xb1f\n\xe1\xbd\x9e}\x13l\x8e\xd3\xfd\xe8u\x08W\x8fh\xd1Y?\xe0 \x12\x88\xbbJ\x92I\t\xa6F\x9cqs>%\x80t\x80\x93\xfb\xbb\xb5\xbb.\xd7\xed\xf4\xbf\x12\x1b\x89\t\xf1\t\xb1PU\xa6\xdc\x00!\x0f\xb6 \x97\xe3\xbc\x9a\xee\x94q\xa2\xe8\x00,r\xfe\xad\x97\xa2\xa1\x05\xd2\x07\xbc\x92 S\xdb\xa3\x1d\xaa\xbd\xb0Nm7\xe0\x8c\xf5\xdd\x12'
DEBUG:localtuya.pytuya:decrypted result='{"dps":{"1":true,"9":0,"17":1,"18":0,"19":0,"20":2338,"21":1,"22":522,"23":25641,"24":13673,"25":3050,"26":0,"38":"memory","39":false,"40":"relay","41":false,"42":"","43":"","44":""}}'
AVAILABLE DPS ARE [{'1': True, '9': 0, '17': 1, '18': 0, '19': 0, '20': 2338, '21': 1, '22': 522, '23': 25641, '24': 13673, '25': 3050, '26': 0, '38': 'memory', '39': False, '40': 'relay', '41': False, '42': '', '43': '', '44': ''}]
INFO:localtuya:COMPLETE response from device ccccccccccccccccc [192.168.1.76].

**** deviceInfo returned OK ****

TuyaDebug (Tuya DPs dump) [1.0.0]

Device ccccccccccccccccc at 192.168.1.76 key xxxxxxxxxxxx protocol 3.3 dev_type type_0a:
    DPS [1] VALUE [True]
    DPS [9] VALUE [0]
    DPS [17] VALUE [1]
    DPS [18] VALUE [0]
    DPS [19] VALUE [0]
    DPS [20] VALUE [2338]
    DPS [21] VALUE [1]
    DPS [22] VALUE [522]
    DPS [23] VALUE [25641]
    DPS [24] VALUE [13673]
    DPS [25] VALUE [3050]
    DPS [26] VALUE [0]
    DPS [38] VALUE [memory]
    DPS [39] VALUE [False]
    DPS [40] VALUE [relay]
    DPS [41] VALUE [False]
    DPS [42] VALUE []
    DPS [43] VALUE []
    DPS [44] VALUE []
```

## Additional information

![obraz](https://user-images.githubusercontent.com/96119906/195925807-5f71c345-b1da-4688-89aa-0a9bf0d4ebec.png)
![obraz](https://user-images.githubusercontent.com/96119906/195925956-0750cd59-f971-4cf8-aca8-40aa0cc64f87.png)
![obraz](https://user-images.githubusercontent.com/96119906/195926004-cf85bc5b-47d4-4776-bbc9-e74f0edfd593.png)
![obraz](https://user-images.githubusercontent.com/96119906/195926591-30e06883-30fa-4e21-b9af-92455c90f6ed.png)

**Comment by MSZ-Spyro** [comment]
Appeard that i forgot that i was using a forked repository for localTuya as my wall thermostat doesnt work properly with the official one :/

**Comment by msavazzi** [comment]
I have the same device and I was not able to connect it with LocalTuya. With the official tuya integration I'm able to get:
<img width="299" alt="image" src="https://user-images.githubusercontent.com/2660032/212469690-35e4ed21-cd79-4d38-a9e8-49bf870b4377.png">

so the switch and the sensors.

Any suggestion on how to get them configured?

**Comment by MSZ-Spyro** [comment]
> I have the same device and I was not able to connect it with LocalTuya.

You need to be more specific. What have you already done ?

**Comment by msavazzi** [comment]
I followed all the steps you did at the beginning and the same screenshots but was not working.

**Comment by MSZ-Spyro** [comment]
> I followed all the steps you did at the beginning and the same screenshots but was not working.

Have you tried tuyadebug and you sure all the DPS ids are the same ? What does your log say ?
